### PR TITLE
feat: rollup to tsup

### PR DESCRIPTION
**Pull request by productdevbook**
https://github.com/egoist/tsup

!Important

UMD Change
```
<script src="https://unpkg.com/@authorizerdev/authorizer-js/dist/index.global.js"></script>
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,17 +18,17 @@ All the above versions require `Authorizer` instance to be instantiated and used
...
 ```
 
-## UMD
+## IIFE
 
 - Step 1: Load Javascript using CDN
 
 ```html
-<script src="https://unpkg.com/@authorizerdev/authorizer-js/lib/authorizer.min.js"></script>
+<script src="https://unpkg.com/@authorizerdev/authorizer-js/lib/index.global.js"></script>
```
Comment (lakhansamani) [inline README.md:31]: Shouldn't this be `dist/index.js`?

**Review by lakhansamani — COMMENTED** [review]
Need confirmation on universal URL

File: README.md (hunk 1 above)
Comment (productdevbook) [reply README.md:31]: no

image

Comment (lakhansamani) [reply README.md]: Still it should `dist/index.global.js` instead of `lib/index.global.js` I suppose?

Comment (productdevbook) [reply README.md]: iam update dist read me and commit.

Comment (lakhansamani) [reply README.md]: Thanks

**Comment by lakhansamani** [comment]
@productdevbook this cloud be breaking change for people using it via CDN as URLs will change!

**Comment by lakhansamani** [comment]
Is there a way we can keep same name for people using it via CDN?

**Comment by productdevbook** [comment]
yes it does, you can exit as v2 when exiting version. Or the next version. People will follow it.

**Comment by lakhansamani** [comment]
Sure 👍

**Pull request merged**